ci/kokoro/docker/build.sh

```diff
@@ -293,11 +293,11 @@ echo "================================================================"
 "${PROJECT_ROOT}/ci/kokoro/docker/upload-coverage.sh" "${docker_flags[@]}"
 
 echo "================================================================"
-#"${PROJECT_ROOT}/ci/travis/dump-logs.sh"
+"${PROJECT_ROOT}/ci/travis/dump-logs.sh"
```

Comment:
I assume that "Remove last Travis build" means the days of ci/travis are numbered?

Reply:
Yes. Some scripts will get refactored, others just removed.